save-backend/src/main/kotlin/com/saveourtool/save/backend/service/LnkExecutionTestSuiteService.kt

```diff
@@ -0,0 +1,54 @@
+package com.saveourtool.save.backend.service
+
+import com.saveourtool.save.backend.repository.LnkExecutionTestSuiteRepository
+import com.saveourtool.save.entities.Execution
+import com.saveourtool.save.entities.LnkExecutionTestSuite
+import com.saveourtool.save.entities.TestSuite
+import org.springframework.stereotype.Service
+
+/**
+ * Service of [LnkExecutionTestSuite]
+ */
+@Service
+class LnkExecutionTestSuiteService(
+    private val lnkExecutionTestSuiteRepository: LnkExecutionTestSuiteRepository,
+) {
+    /**
+     * @param execution
```

Comment:
Here and below: I suggest to either document the parameter, or add `@Suppress("KDOC_WITHOUT_PARAM_TAG")`.

Reply:
Fixed KDoc comment